src/main/pages/che-7/contributor-guide/proc_adding-support-for-a-new-language.adoc

```diff
@@ -1,27 +1,32 @@
 [id="adding-support-for-a-new-language_{context}"]
 = Adding support for a new language
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section describes how to add support for a new language into Eclipse Che. 
 
```

Comment:
Support for a new language is a VS Code plugin that uses the languageserver-node library to hook into the VS Code API.

Reply:
@MichalMaler VS Code doesn't have plug-ins it has extensions. And this section is about how to contribute new Theia plug-in to support new language. The information about `languageserver-node` library is in the paragraph below.

Reply:
@svor I see it now, my apologies. All good :)